shaded/core/build.gradle.kts

```diff
@@ -69,7 +69,7 @@ tasks {
         listOf(
             "com.fasterxml.jackson",
             "com.google.protobuf",
-            "org.apache.http",
+            "org.apache.hc",
```

Comment:
Could we add a test for it? (Apache Arrow has this test for their FlightSQL JDBC driver -> https://github.com/apache/arrow/blob/00e7c65e17f7d59f7c9954473b15b8ffae8dfd1a/java/flight/flight-sql-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ITDriverJarValidation.java#L114